### ISSUE TITLE
Usar algoritmo de colocación de plantas

### ISSUE THREAD

**Issue by aernesto** [aernesto/Lux-Design-S2#3]
A ver si logramos usar el algoritmo en el método `early_setup()` de un agente.

**Comment by aernesto** [comment]
@AlfonsoOC , en lo que estás probando, mencionan Gaussian Mixture Models de casualidad?
Voy a experimentar tantito con [esto](https://scikit-learn.org/stable/auto_examples/mixture/plot_gmm_pdf.html)

**Comment by AlfonsoOC** [comment]
no, ese es otro metodo que tambien me llamo la atencion pero me esta costando trabajo usando manhattan distances y no he querido meterme a otra cosa sin terminar lo primero, se ve interesante tambien.
